src/vue/forms/TfaForm.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <form
+    class="app-form tfa-form"
+    @submit.prevent="isFormValid() && enableTfa()"
+  >
+    <template v-if="!form.factor">
+      <div class="app__form-row">
+        <div class="app__form-field">
+          <input-field
+            v-model="form.password"
+            @blur="touchField('form.password')"
+            id="tfa-password"
+            type="password"
+            :error-message="getFieldErrorMessage('form.password')"
+            :label="'tfa-form.password-lbl' | globalize"
+            :disabled="formMixin.isDisabled"
+          />
+        </div>
+      </div>
+      <div class="app__form-actions">
+        <button
+          v-ripple
+          type="button"
+          class="tfa-form__btn"
+          @click="toggleFactor"
+          :disabled="formMixin.isDisabled"
+        >
+          <template v-if="isTotpEnabled">
+            {{ 'tfa-form.disable-btn' | globalize }}
+          </template>
+          <template v-else>
+            {{ 'tfa-form.next-btn' | globalize }}
+          </template>
+        </button>
+      </div>
+    </template>
+    <template v-else>
+      <p class="tfa-form__hint">
+        {{ 'tfa-form.qr-desc' | globalize }}
+      </p>
+      <key-viewer
+        :value="form.factor.seed"
+        :is-clipboard-shown="false"
+      />
+      <p class="tfa-form__hint">
+        {{ 'tfa-form.secret-code-desc' | globalize }}
+      </p>
+      <clipboard-field
+        :value="form.factor.secret"
+        :label="'tfa-form.secret-lbl' | globalize"
+      />
+      <p class="tfa-form__hint tfa-form__code-hint">
+        {{ 'tfa-form.tfa-code-desc' | globalize }}
+      </p>
+      <div class="app__form-field">
+        <input-field
+          v-model="form.code"
+          @blur="touchField('form.code')"
+          id="tfa-code"
```

Comment:
why do we need an id here?

Reply:
https://github.com/tokend/web-client/pull/38#discussion_r253804830